### PR TITLE
Fix bot spawning with dog model

### PR DESCRIPTION
**Pull request by agustinesco**
The bots will spawn with a random character defined in the config file now

### CODE REVIEW

File: client/Assets/Scenes/Battle.unity
Comment (BertovDev) [inline client/Assets/Scenes/Battle.unity:1]: You dont need to change the `Battle` scene. Change already did in the `CameraUI` prefab

`git checkout main -- client/Assets/Scenes/Battle.unity`

**Review by BertovDev — CHANGES_REQUESTED** [review]

**Review by AminArria — CHANGES_REQUESTED** [review]
Update `mix.lock` with `mix deps.get lamda_game_engine` once the other PR is merged

**Review by AminArria — APPROVED** [review]
LGTM (only looked at backend part)

File: client/Assets/Scenes/Battle.unity
Comment (agustinesco) [reply client/Assets/Scenes/Battle.unity:1]: Sure! i'm not sure what happened here

**Review by BertovDev — CHANGES_REQUESTED** [review]
If dog prefab is deleted and not used anymore their assets should also have to be removed.

you can find them in `client/Assets/ThirdParty/DogKnight` I would delete the whole folder.

**Comment by agustinesco** [comment]
> If dog prefab is deleted and not used anymore their assets should also have to be removed.
> 
> you can find them in `client/Assets/ThirdParty/DogKnight` I would delete the whole folder.

Sure thing, i deleted the folder and everything seems to be working correctly

**Review by AminArria — APPROVED** [review]

**Pull request merged**